static/docs/commands-reference/import-url.md

```diff
@@ -119,195 +119,155 @@ downloaded file or directory from the external data source.
 
 ## Examples
 
-To illustrate the examples we will be using the project used in _Get Started_.
-It is very easy to setup a playground for the examples below. It's completely
-optional though.
+To illustrate the examples we will be using the project explained in the
+[Get Started](/doc/get-started) section. With it, it's very easy to setup a
+playground for the examples below.
 
 <details>
 
-### Click and expand to setup the project
+### Click and expand to setup the sample project
 
-This step is optional, and you can run it only if you want to run this examples
-in your environment. First, you need to download the project:
-
-```dvc
-    $ git clone https://github.com/iterative/example-get-started
-```
-
-Second, let's install the requirements. But before we do that, we **strongly**
-recommend creating a virtual environment with
-[virtualenv](https://virtualenv.pypa.io/en/stable/) or a similar tool:
+Follow these instructions before each example if you actually want to try them
+on your system. First, download the project and move into it:
```

Comment:
`move into it` sounds a it weird :)

Reply:
OK how about "`cd` into it"? Also updated in 1a214ea